tests/ui/too_long_first_doc_paragraph_module_level.rs

```diff
@@ -0,0 +1,10 @@
+#![warn(clippy::too_long_first_doc_paragraph)]
```

Comment:
Can we put this test in `tests/ui/too_long_first_doc_paragraph-fix.rs`?

Reply:
Yes